### PR TITLE
add APK size benchmark for Flutter Gallery app and a card

### PR DESCRIPTION
**Pull request by yjbanov**
/cc @devoncarew

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -183,6 +183,24 @@
...
+      <div class="card-inside">
+        <div class="metric-type">App size <span class="metric-name-wrapper">(<span class="metric-name"></span>)</span></div>
+        <div class="metric-data"><span class="metric-number"></span><span class="metric-unit">kb</span></div>
+
+        <table class="stats-table">
+          <tr>
+            <td class="stats-value"><span class="metric-target">10,000</span>kb</td>
+          </tr>
+          <tr>
+            <td class="stats-label">Q2 goal</td>
```
Comment (devoncarew) [inline web/index.html]: Still a Q2 goal?

**Comment by devoncarew** [comment]
lgtm!

If we don't have, we should file a separate issue for measuring IPA sizes, as they might track ~independently from APKs.

File: web/index.html (hunk 1 above)
Comment (yjbanov) [inline web/index.html]: Well, we still have 2 weeks to fix it :)

**Comment by yjbanov** [comment]
> If we don't have, we should file a separate issue for measuring IPA sizes, as they might track ~independently from APKs.

Filed https://github.com/flutter/flutter/issues/4609. I wanted to include IPA in this PR but there are still unresolved issues with `flutter build ios --release`.

**Pull request merged**